src/Microsoft.Azure.WebJobs.Host/Loggers/UpdateOutputLogCommand.cs

```diff
@@ -139,7 +139,10 @@ public void Dispose()
                     _synchronizedWriter.Flush();
                 }
 
-                snapshot = _innerWriter.ToString();
+                // Explicitly specify the length. Without this, any writes occurring 
```

Comment:
>// Explicitly specify the length. Without this, any writes occurring  [](start = 16, length = 69)

This is already under a lock; and we have a synchronized writer.    Do we still need those?